runtime/port/unix/j9sysinfo.c

```diff
@@ -1561,6 +1567,10 @@ getCacheTypes(struct J9PortLibrary *portLibrary,
 	const int32_t cpu, const int32_t level)
 {
 	int32_t result = J9PORT_ERROR_SYSINFO_NOT_SUPPORTED;
+#if defined(J9OS_I5_V6R1)
+        /*getsystemcfg() doesn't supported on i 7.1, return directly.*/
```

Comment:
The ifdef is for V6R1 but the comment says `7.1`.  Something seems off here.

Reply:
It caused by our code management way. On IBM i, we skip ship all jdks on i 7.1.  So use  J9OS_I5_V6R1 for 7.1.  You should not see anything like J9OS_I5_V7R1.

Reply:
@DanHeidinga  FYI. Pushed new changes.